src/dotnet/CarbonAware.WebApi/CarbonAware.WebApi.csproj

```diff
@@ -19,11 +19,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.14.0" />
```

Comment:
Same comment as above

Reply:
Was thinking similarly to these Qs from Juan -- what was the impetus to move to the new(er) versions? known issues? new features we need? other?